src/pages/Home.module.css

```diff
@@ -6,6 +6,25 @@
   box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
 }
 
+.startButton {
```

Comment:
We should really have renamed this file to ScenarioList.module.css and created a separate one for the Splash Screen